ui/src/views/Activate/Syncs/SyncForm/FinaliseSync/CronScheduleType.tsx

```diff
@@ -0,0 +1,114 @@
+import { Box, Input, Text } from '@chakra-ui/react';
+import { parseExpression } from 'cron-parser';
+import { isValidCron } from 'cron-validator';
+import { FormikContextType } from 'formik';
+
+import dayjs from 'dayjs';
```

Comment:
Why are we using `dayjs` here when we are already using Moment? Is this not possible with Moment?

Reply:
dayjs works well with timezones.
moment i have faced lot of issues in displaying time with timezones.

we will create a tech debt to remove moment and use dayjs across